widget/playground/src/components/MultiList/MultiList.tsx

```diff
@@ -136,31 +132,13 @@ export function MultiList(props: MultiListPropTypes) {
           <Divider size={20} />
         </>
       )}
-      <TextField
+      <SearchInput
         onChange={(e) => setSearchValue(e.target.value)}
```

Comment:
I think it's not necessary to use both "onChange"  and "setValue" and maybe it's better to handle the event in the child component "SearchInput"

Reply:
you'll right, It fixed .